lib/spack/spack/build_systems/cmake.py

```diff
@@ -101,13 +114,29 @@ def std_cmake_args(self):
     def _std_args(pkg):
         """Computes the standard cmake arguments for a generic package"""
         try:
+            generator = pkg.generator
+        except AttributeError:
+            generator = 'Unix Makefiles'
+
+        # Make sure a valid generator was chosen
+        valid_generators = ['Unix Makefiles', 'Ninja']
+        if generator not in valid_generators:
+            msg  = "The following CMake generator is not yet supported "
```

Comment:
wouldn't it be better to list the valid generator names instead of stating that the one chosen is not yet supported?